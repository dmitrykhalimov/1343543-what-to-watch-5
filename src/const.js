export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
