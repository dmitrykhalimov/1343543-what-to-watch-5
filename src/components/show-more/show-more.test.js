import React from "react";
import renderer from "react-test-renderer";
import ShowMore from "./show-more";
import {TEST_MOCKS} from "../../const";

describe(`ShowMore`, () => {
  it(`Should ShowMore render correctly isShown`, () => {
    const tree = renderer
      .create(
          <ShowMore
            rendered={1}
            filmsQuantity={1}
            onShowMore={TEST_MOCKS.noop}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
