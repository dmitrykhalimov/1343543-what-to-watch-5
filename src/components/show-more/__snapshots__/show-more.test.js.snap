// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMore Should ShowMore render correctly isShown 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
